gnd/src/main/java/com/google/android/gnd/repository/ProjectRepository.java

```diff
@@ -131,6 +131,10 @@ public void activateProject(String projectId) {
         .compose(Loadable::loadingOnceAndWrap);
   }
 
+  public Flowable<Loadable<List<Project>>> getSavedProjects() {
```

Comment:
`getOfflineProjectsOnceAndStream()`, as per https://github.com/google/ground-android/wiki/RxJava-Usage-Guidelines#observable-and-flowable